### ISSUE TITLE
ValueError: not enough values to unpack (expected 1, got 0)

### ISSUE THREAD

**Issue by ms3534** [py-pdf/PyPDF2#1145]
```python
import os
from PyPDF4 import PdfFileReader, PdfFileMerger
from PIL import Image
from PIL import ImageFile
import argparse
import string
import collections

def png_to_pdf():
    merger = PdfFileMerger()
    merger.append(PdfFileReader(open('C:\file-0.pdf', 'rb')))

def main():
    png_to_pdf()

main()
```

![error](https://user-images.githubusercontent.com/104185732/180363254-f37ec61b-6869-4896-b49f-3586fc2193d7.JPG)
[file-0.pdf](https://github.com/py-pdf/PyPDF2/files/9164743/file-0.pdf)

**Comment by MartinThoma** [comment]
@ms3534 You've shared code that uses `PyPDF4`. Does the issue also appear with PyPDF2? Which version of PyPDF2 have you used for testing?

**Comment by ms3534** [comment]
@MartinThoma: Yeah, the same issue is coming in PyPDF2

**Comment by ms3534** [comment]
![image](https://user-images.githubusercontent.com/104185732/180373337-3e466176-a9bf-40b0-bf2f-b8953be5d28f.png)

**Comment by MartinThoma** [comment]
Could you please execute

```
import PyPDF2
print(PyPDF2.__version__)
```

And share the results?

**Comment by MartinThoma** [comment]
Oh, never mind, @pubpub-zz already confirmed that the issue also exists for a recent version :-)

**Comment by pubpub-zz** [comment]
@ms3534, 
The  problem is coming from a /FitB in a named destination with null parameters. Although accepted by the spec, it is very rare.

a fix is proposed in #1152 
Also When I did further test I've noticed that with current version the text extraction on 4th page is not good (spaces) Thanks for this new sample 😀
